src/auth/Fingerprint.cpp

```diff
@@ -169,7 +169,14 @@ void CFingerprint::handleVerifyStatus(const std::string& result, bool done) {
                 m_sFailureReason = "Fingerprint auth disabled (too many failed attempts)";
             } else {
                 done = false;
-                startVerify(true);
+                g_pHyprlock->addTimer(
+                    std::chrono::milliseconds(**(Hyprlang::INT* const*)(g_pConfigManager->getValuePtr("auth:fingerprint:retry_delay"))),
+                    [](std::shared_ptr<CTimer> self, void* data) {
+                        ((CFingerprint*)data)->startVerify(true);
+                        g_pAuth->m_bDisplayFailText = false;
```

Comment:
Why do you to set this to false?

Reply:
I'm honestly not sure. I think I was testing something and forgot to remove it.